lib/student.rb

```diff
@@ -1,9 +1,12 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    @classroom.add_student(self)
   end
```

Comment:
- [ ] Since associations are usually implemented with a separate method I'd suggest you create a distinct method for adding a classroom to a student instead of implementing it in the constructor. This will especially come in handy when you start to write unit tests in the coming milestone. 😉